frame/transaction-payment/src/lib.rs

```diff
@@ -56,7 +56,7 @@ use sp_runtime::{
 };
 use pallet_transaction_payment_rpc_runtime_api::RuntimeDispatchInfo;
 
-type Multiplier = Fixed128;
+type Multiplier = FixedI128;
```

Comment:
```suggestion
pub type Multiplier = FixedI128;
```

and then we can use it all over `bin/node/runtime/src/impls.rs` if you find it suitable.

Reply:
I'm not sure,  is there a better place to define `Multiplier`?

Reply:
Is it ok to call it `FeeMultiplier` instead?